### PR TITLE
Merge master into re feature branch

### PR DESCRIPTION
**Pull request by chbatey**
Worth looking at Shard.scala and ShardCoordinator.scala all other changes were either conflict free or conflicting on the sort imports PR

### CODE REVIEW

File: akka-cluster-sharding/src/main/scala/akka/cluster/sharding/ShardCoordinator.scala [hunk 1]
```diff
@@ -1150,8 +1149,10 @@ private[akka] class DDataShardCoordinator(
     case additional   => WriteMajorityPlus(settings.tuningParameters.waitingForStateTimeout, majorityMinCap, additional)
   }
 
-  implicit val node = Cluster(context.system)
-  private implicit val selfUniqueAddress = SelfUniqueAddress(node.selfUniqueAddress)
+  // FIXME make the all shards consistency is reflected in the ddata remembered thingy
```
Comment (johanandren) [inline akka-cluster-sharding/src/main/scala/akka/cluster/sharding/ShardCoordinator.scala:1152]: So we deal with the consistency improvements for the coordinator separately?

**Review by johanandren — COMMENTED** [review]
Looked at coordinator and shard but didn't really see any changes?

File: akka-cluster-sharding/src/main/scala/akka/cluster/sharding/ShardCoordinator.scala (hunk 1 above)
Comment (chbatey) [reply akka-cluster-sharding/src/main/scala/akka/cluster/sharding/ShardCoordinator.scala:1152]: They'd already been ported but i wasn't sure when merging which sets of consistencies were remembered entities (majority) vs the new ones (all or majority plus).

Patrik had ported that PR to this branch so you're right there aren't many changes

**Comment by chbatey** [comment]
Seems i missed the change that fixed the rest, running validation again!

**Review by johanandren — APPROVED** [review]
Green, quickly, merge!

**Pull request merged**